phpunit.xml.dist

```diff
@@ -23,6 +23,7 @@
             <exclude>
                 <directory>./tests</directory>
                 <directory>./vendor</directory>
+                <file>./src/NullCache.php</file>
```

Comment:
Why?

Reply:
Do we need to add tests for it?

Reply:
Why not? https://github.com/yiisoft/cache/blob/7b6be46688f83e272972f61e00ac22a1d6dd0332/tests/NullCacheTest.php

Reply:
I didn't notice it.